contrib/win32/install/install/README.md

```diff
@@ -0,0 +1,54 @@
+# Windows OpenSSH - MSI project
+
+The project, based on the [WiX Toolset](https://wixtoolset.org/), creates a Windows MSI package to install OpenSSH on Windows computers.
+
+The projects supports building MSI packages for 32bit and 64bit platforms. In addition the MSI supports a wider range of installation flags to perform the installation of OpenSSH complete silently. Furthermore the MSI ensures that Debug end Release Versions cannot installed simultaneously. When the OpenSSH server components are installed, the MSI will also configure the Windows Firewall, to ensure that the SSH Server is reachable over the network.
+
+## Build Parameters
+
+The following parameters are supported 
+
+* `MANUFACTURER`: (optional) the manufacturer of the MSI; default: Microsoft Corporation
+* `CONFIGURATION`: (required) the configuration used; Valid values: `Release`, `Debug`
+* `VERSION`: (optional) the version tag of the MSI; default: `0.0.0.0`
+* `SOURCEDIR`: (required) the path to the input files
+* `PRODUCTID`: (optional) the Product GUID of the created MSI; default: dynamic (`*`), changes every time the MSI is build
+
+The parameters are passed via `Msbuild` properties.
+
+Example:
+
+```
+$ msbuild openssh-install.wixproj /p:Version=8.1.1 /p:Configuration=Release /p:Platform=x64 /p:SourceDir=c:\openssh-portable\bin /p:ProductId="15512e42-e946-45f9-bbe5-a8e495bd2e2b"
+```
+
+## MSI internals
+
+### Upgrade Codes
+
+* Debug Version: `c7a00fb1-c477-40d3-a951-ec4c749da439`
```

Comment:
Good point, but isn't Windows INTEL/AMD the only plattform which has this issue because there is this infamous WOW64 😁  and @SteveL-MSFT do we have to support 32bit Windows anyway?

Reply:
Windows Installer actually differentiates between x86 and x64, disabling WOW64 virtualization in almost all cases. You can't assume it with installer packages.

Reply:
Removed the debug version stuff